metanorma-standoc.gemspec

```diff
@@ -31,7 +31,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency "addressable", "~> 2.8.0"
   spec.add_dependency "asciidoctor", "~> 2.0.0"
   spec.add_dependency "crass", "~> 1.0.0"
-  spec.add_dependency "iev", "~> 0.3.0"
+  spec.add_dependency "iev", ">= 0.3.4"
```

Comment:
You can keep this as ~> 0.3.4 which means larger or equal to 0.3.4 but not above 0.3.x.